java/pom.xml

```diff
@@ -158,6 +158,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <junit.version>5.4.2</junit.version>
         <ai.rapids.refcount.debug>false</ai.rapids.refcount.debug>
+        <ai.rapids.allow.nonempty.nulls>true</ai.rapids.allow.nonempty.nulls>
```

Comment:
`nulls` should stay first, as we are talking about null. So this should be `nulls.noempty` instead.

Reply:
Or, this should be better: `ai.rapids.allow.nonempty_nulls`.